pyproject.toml

```diff
@@ -42,5 +45,21 @@ Documentation = "https://astrodb-utils.readthedocs.io/en/latest/"
 root = '.'
 
 [tool.pytest.ini_options]
-testpaths = ["tests"]
+testpaths = ["astrodb_utils/tests"]
 norecursedirs = "astrodb-template-db"
+
+[tool.ruff.lint]
+# Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
+# Unlike Flake8, Ruff doesn't enable pycodestyle warnings (`W`) or
+# McCabe complexity (`C901`) by default.
+# See https://docs.astral.sh/ruff/rules/
+select = ["E4", "E7", "E9", "F", "I", "PL", "C901", "RUF010"]
+#select = ["ALL"]
+ignore = ["PLR"]  # pylint refactor
```

Comment:
I took a stab at some useful linter warnings, but we can expand/restrict this as needed.